stock_push_delay/models/stock_move.py

```diff
@@ -0,0 +1,17 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockMove(models.Model):
+
+    _inherit = "stock.move"
+
+    def _push_apply(self):
+        """Manual triggering"""
+        if self.env.context.get("manual_push", False):
+            new_move = super(StockMove, self)._push_apply()
```

Comment:
```suggestion
            new_move = super()._push_apply()
```

Reply:
I'm wondering if this behavior shouldn't be activated through option as this changes standard behavior.

@LoisRForgeFlow @AaronHForgeFlow

Reply:
The option will install the module? or maybe company specific? For me it is ok.